src/binder/inc/clrprivbinderassemblyloadcontext.h

```diff
@@ -45,14 +67,30 @@ class CLRPrivBinderAssemblyLoadContext : public IUnknownCommon<ICLRPrivBinder>
             /* [out] */ HRESULT *pResult,
             /* [out] */ ICLRPrivAssembly **ppAssembly);
 
+#ifdef FEATURE_COLLECTIBLE_ALC
+
+    //=========================================================================
+    // IAssemblyLoadContext functions
+    //-------------------------------------------------------------------------
+    STDMETHOD(GetLoaderAllocator)(
+        /* [retval][out] */ LoaderAllocator** pLoaderAllocator);
+
+#endif // FEATURE_COLLECTIBLE_ALC
+
 public:
     //=========================================================================
     // Class functions
     //-------------------------------------------------------------------------
 
-    static HRESULT SetupContext(DWORD      dwAppDomainId, CLRPrivBinderCoreCLR *pTPABinder, 
-                                UINT_PTR ptrAssemblyLoadContext, CLRPrivBinderAssemblyLoadContext **ppBindContext);
-                    
+    static HRESULT SetupContext(DWORD      dwAppDomainId,
+                                CLRPrivBinderCoreCLR *pTPABinder,
+                                LoaderAllocator* pLoaderAllocator,
+                                void* loaderAllocatorHandle,
+                                UINT_PTR ptrAssemblyLoadContext,
+                                CLRPrivBinderAssemblyLoadContext **ppBindContext);
+
+    static void DestroyContext(CLRPrivBinderAssemblyLoadContext *pBindContext, INT_PTR ptrManagedStrongAssemblyLoadContext);
```

Comment:
Why is this a static and not an instance method, given that we will be passing the ALC* to it as an argument?